### DISCUSSION TITLE
Option to Decrease Minimum Volume

### DISCUSSION THREAD

**Discussion by CodingTil** [ciderapp/Cider#1342] [Feature Request]
Even setting the minimum volume to the lowest volume, that is note mute, I can only set my OS (ubuntu) volume to 50%, otherwise my ears will die of the loud volume.

This gets really annoying, when using other applications, that have normal volume defaults.

Adding an option to decrease the minimum volume, would really help, while preserving the usage for people without this issue. This could be implemented similarly to the Option to Increase the Maximum Volume.

I did not only find the default minimum volume way to high on ubuntu, but also back on windows.

**Comment by maikirakiwi** [comment]
Don't use the mouse scroll wheel
